rfi_file_monitor/utils/widgetparams.py

```diff
@@ -70,6 +43,10 @@ def _spinbutton_value_changed_cb(self, spinbutton: Gtk.SpinButton, param_name: s
     def _combobox_changed_cb(self,combobox: Gtk.ComboBoxText, param_name: str):
         self._params[param_name] = combobox.get_active_text()
 
+    @final
+    def _button_clicked_cb(self, button: Gtk.Button, param_name:str):
```

Comment:
I don't see how you can use a button to back a param that can be used in the pipeline... this is for the dynamic combobox I guess?

Reply:
removed this now as it was not correct